ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/TemporaryEngine.java

```diff
@@ -161,7 +161,9 @@ protected void after() {
                 SCHEMA,
                 Optional.empty(),
                 false,
-                topic
+                topic,
+                false,
+                false
```

Comment:
What is `TemporaryEngine` and would we need to add support for source tables?

Reply:
We'll add it in a follow-up PR. This `TemporaryEngine` is used by the `ListSourceExecutorTest`, which tests the DESCRIBE and SHOW commands. There's a JIRA to make sure DESCRIBE correctly handles source streams & tables.